File: compiler/testdata/ruby/test_case_expr.rb
def describe(n)
  result = case n
  when 1 then "one"
  when 2 then "two"
  else "other"
  end
  puts result
end

def categorize(n)
  case n
  when 1 then "small"
  when 2 then "medium"
  else "large"
  end
end

describe(1)
categorize(1)

File: compiler/testdata/go/test_case_expr.go
package main

import "fmt"

func Describe(n int) {
	var result string
	switch n {
	case 1:
		result = "one"
	case 2:
		result = "two"
	default:
		result = "other"
	}
	fmt.Println(result)
}
func Categorize(n int) string {
	switch n {
	case 1:
		return "small"
	case 2:
		return "medium"
	default:
		return "large"
	}
}
func main() {
	Describe(1)
	Categorize(1)
}
